istqb-ctel-itp-atp-complete-guide.md

```
@@ -0,0 +1,1783 @@
+- テストコストの削減（ROI改善）
+- 予測可能性・報告能力の向上
+- サプライヤー要件への対応（例：TMMi Level 3以上を要求される）
+
+組織的ドライバー：
+- ソフトウェア障害によるビジネス損失の経験
+- 第三者機関へのアウトソーシング展開
+- 規制・コンプライアンス要件（例：FDA、SOX法）
+
+改善の文脈（Context）：
+テストプロセス改善は以下の文脈の中で行われる：
+```mermaid
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)